packages/blob/src/index.ts

```diff
@@ -324,7 +327,13 @@ function shouldFetchClientToken(
 }
 
 function getApiVersionHeader(): { 'x-api-version'?: string } {
-  const versionOverride = process.env.VERCEL_BLOB_API_VERSION_OVERRIDE;
+  let versionOverride = null;
+  try {
+    // wrapping this code in a try/catch as this function is used in the browser and Vite doesn't define the process.env.
+    // As this varaible is NOT used in production, it will always default to the BLOB_API_VERSION
+    process.env.VERCEL_BLOB_API_VERSION_OVERRIDE ||
```

Comment:
```suggestion
    versionOverride = process.env.VERCEL_BLOB_API_VERSION_OVERRIDE ||
```

right? cc @correttojs